infra/presubmit.py

```diff
@@ -346,9 +360,13 @@ def main():
     success = check_license(changed_files)
     return bool_to_returncode(success)
 
+  if args.command == 'test':
+    return run_tests()
+
   # Otherwise, do all of them.
+
   success = do_checks(changed_files)
-  return bool_to_returncode(success)
+  return run_tests() and bool_to_returncode(success)
```

Comment:
it does look like there is a failure in Travis right now, which it is not complaining about: https://travis-ci.org/google/oss-fuzz/jobs/652585782

Reply:
otherwise the change looks good and I'm ready to LGTM, but need to clarify these + fix the nits I left above

Reply:
The tests need to be run as root because of file operations/tempfile. I updated the script to run the unit tests as root.